js/screens/SignUpName/SignUpNameScreen.js

```diff
@@ -23,27 +24,48 @@ class SignUpNameScreen extends Component {
     this.props.dispatch(fetchLastName(text));
   };
 
+  handleNextScreen = () => {
+    const { navigation, firstName, lastName } = this.props;
+    const nameError = {
+      code: 'Missing Name',
+      message: 'Please submit a first and last name'
```

Comment:
this error object should go to the errorMessages.js file (the one mentioned above).